src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -58,11 +58,7 @@ public bool CheckBoxesEnabled
             }
             set
             {
-                _checkBoxesEnabled = value;
-                _updateButtonContainer.Visibility =
-                    _checkBoxesEnabled ?
-                    Visibility.Visible :
-                    Visibility.Collapsed;
+                _checkBoxesEnabled = value;                
```

Comment:
UpdateCheckBoxStatus() is always called whether CheckBoxesEnabled is true or false.

The thing is that the bar. i.e. update button container, can be invisible even if CheckBoxesEnabled is true. E.g. when there are not updates, then when user selects "Updates" tab, CheckBoxesEnabled will be set to true. But since there are no updates, the bar should be invisible.